daffodil-japi/src/main/scala/org/apache/daffodil/japi/packageprivate/Utils.scala

```diff
@@ -82,6 +84,8 @@ private[japi] object ValidationConversions {
       case SValidationMode.Off => ValidationMode.Off
       case SValidationMode.Limited => ValidationMode.Limited
       case SValidationMode.Full => ValidationMode.Full
+      // todo;; support custom in java?
+      case _ => ValidationMode.Off
```

Comment:
Not sure how to approach this

Reply:
JAPI modeFromScala might not be used at all. I searched and didn't find any usages. The whole ValidationConversions object is package private. I think modeFromScala can probably be removed.

Reply:
> I think modeFromScala can probably be removed.

done